src/openepd/model/specs/generated/electrical.py

```diff
@@ -293,7 +293,26 @@ class LightingV1(BaseOpenEpdHierarchicalSpec):
 class ElectricalConduitV1(BaseOpenEpdHierarchicalSpec):
     """Tubing used to protect and route electrical wiring in a building or structure."""
 
-    _EXT_VERSION = "1.0"
+    _EXT_VERSION = "1.1"
+
+    nominal_diameter: LengthMmStr | None = pyd.Field(
+        default=None,
+        description="Nominal Diameter is also known as the mean or average outside diameter.",
+        example="100 mm",
+    )
+    outer_diameter: LengthMmStr | None = pyd.Field(
+        default=None,
+        description="The measurement of the distance of a straight line from one point on the outer wall of the pipe.",
+        example="100 mm",
+    )
+    inner_diameter: LengthMmStr | None = pyd.Field(
+        default=None,
+        description="The measurement of the distance of a straight line from one point on the inner wall of the pipe.",
```

Comment:
This sentence looks incomplete - from one point on the inner wall to where? to the opposite wall? Or it is as is ?

Reply:
Also, let's be consistent with the full stops at the end of description.

Reply:
> This sentence looks incomplete - from one point on the inner wall to where? to the opposite wall? Or it is as is ?

to other point of inner wall. 

See scheme:

<img width="586" alt="image" src="https://github.com/user-attachments/assets/04fd2d8e-a462-471c-b70e-c6989fd3084d">

Reply:
Slightly change the phrase